enclave_mgmt/compile_models.py

```diff
@@ -24,6 +25,9 @@
 MODEL_QUIZ_ATTEMPTS = "quiz_attempts.csv"
 MODEL_QUIZ_ATTEMPT_MULTICHOICE_RESPONSES = \
      "quiz_attempt_multichoice_responses.csv"
+MODEL_CONTENT_LOADS = "content_loads.csv"
+MODEL_IB_PROBLEM_ATTEMPTS = "ib_problem_attempts.csv"
```

Comment:
@P-Gill97 - I think this CSV output name needs to be updated to reflect the change `usage.md` (here and in the tests). It might also be nice to patch instances of `ib_problem_attempts` to `ib_pset_problem_attempts` in the code for consistency.